pkg/minikube/minikube.go

```diff
@@ -154,6 +155,32 @@ func checkForExistingCluster() error {
 		fmt.Scanf("%s", &resp)
 		if strings.ToLower(resp) != "y" {
 			fmt.Println("Installation skipped")
+			checkKnativeNamespace := exec.Command("kubectl", "get", "namespaces")
+			output, err := checkKnativeNamespace.CombinedOutput()
+			namespaces := string(output)
+			if err != nil {
+				fmt.Println(string(output))
+				return fmt.Errorf("check existing cluster: %w", err)
+			}
+			if strings.Contains(namespaces, "knative") {
+				fmt.Print("Knative installation already exists.\nDelete and recreate the cluster [y/N]: ")
+				fmt.Scanf("%s", &resp)
+				if strings.ToLower(resp) != "y" {
+					fmt.Println("Skipping installation")
+					installKnative = false
+					return nil
+				} else {
+					fmt.Println("deleting cluster...")
+					deleteCluster := exec.Command("minikube", "delete", "--profile", "--insecure-registry", clusterName)
+					if err := deleteCluster.Run(); err != nil {
+						return fmt.Errorf("delete cluster: %w", err)
+					}
+					if err := createNewCluster(); err != nil {
+						return fmt.Errorf("new cluster: %w", err)
+					}
```

Comment:
let's pull this into a separate function to use here and lines 187-93

Reply:
> let's pull this into a separate function to use here and lines 187-93

still need this one

Reply:
Right, will add it today.